proto/buf/alpha/registry/v1alpha1/organization.proto

```diff
@@ -278,6 +282,20 @@ message AddOrganizationGroupRequest {
 
 message AddOrganizationGroupResponse {}
 
+message UpdateOrganizationGroupRequest {
+  // The ID of the organization to which this group belongs.
+  string organization_id = 1;
+  // The name of the group to update.
+  string group_name = 2;
+  // The role to associate with this organization group.
+  //
+  // Setting this to 'UNSPECIFIED' will remove the override. Leaving this unset will not update this
```

Comment:
Should this error if it's unset, make it mandatory either explicitly UNSPECIFIED or the desired updated role override?

Reply:
Generally, the mutable fields in an Update request are optional so that users don't have to update everything in one shot, they can update just the one thing they care about. It would be more obvious in this case if there was more than one thing to update, but I didn't want to deviate from that style. See https://github.com/bufbuild/registry-proto/blob/2f065e825828d75db601227b33d824a952b9363e/buf/registry/owner/v1/user_service.proto#L164-L180 for example.